commands/show/nodepool/command.go

```diff
@@ -134,13 +134,13 @@ func fetchNodePool(args arguments) (*result, error) {
 		return nil, microerror.Mask(err)
 	}
 
-	res.instanceTypeDetails, err = awsInfo.GetInstanceTypeDetails(res.nodePool.NodeSpec.Aws.InstanceType)
-	if err != nil {
-		return nil, microerror.Mask(err)
-	}
+	var nodespecErr error
+	res.instanceTypeDetails, nodespecErr = awsInfo.GetInstanceTypeDetails(res.nodePool.NodeSpec.Aws.InstanceType)
```

Comment:
Why can't we keep that simple and call it `err`?

Reply:
Don't know. Got confused by a compiler message before. Changing.